### ISSUE TITLE
Beta platform screen reader accessibility issues

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/freeCodeCamp#16265]
This accessibility feedback comes from Roger Newell, who has been working through www.freeCodeCamp.org using a screen reader fine, but is having a lot of trouble with beta.freecodecamp.org. Here is his feedback verbatim. We are looking for an accessibility-minded contributor to help improve these things. 

He's discovered these issues using both the Windows screen reader NVDA and JAWS with Internet Explorer on Windows 10. Again, these are all his words:

1. Activating the "map", "step" and "community" buttons appears to do nothing about 99% of the time, and the other 1% of the times I tried, it takes me to the first lesson.

2. I am not an accessibility expert by any means, but perhaps someone with a little more knowledge in the field can check the "community" button. My screen reader announces that it is a "button menu", and the only times I have ever heard that description for a button on modern webpages have been on websites that are inaccessible. I am not saying that this button is inaccessible, but perhaps someone should just check on it?

3. When I see the types of challenges, activating a link such as "front-end libraries" for example does not seem to do anything apart from my screen reader announcing that the link is now a "visited link" and the page reloads.

4. When I am in the first part of the first challenge, pressing "go to my next step" does not do anything either.

5. When you search for something use the search field, the results (assuming that they do display) also cannot be seen with either screen reader.

6. Using both of the aforementioned screen readers, the words in the links for the different types of challenges are spoken with hyphens between each word such as "data-visualization". Even if this is how they appear in print, it is not considered correct accessibility practice as it sounds rather awkward to hear "data dash visualization" which is how a screen reader would normally pronounce such a string. 

7. Finally,WAI-ARIA landmarks are not used as often as they should be on these pages, and at least in the first challenge, there is no heading to separate the map from the content of the lesson--there are headings in the current version of the website.

**Comment by QuincyLarson** [comment]
OK - so @inscriptioelectronicaaustralia tried using freeCodeCamp in Chrome, and he said virtually all of these issues are resolved, except for the issues explained in https://github.com/freeCodeCamp/freeCodeCamp/issues/12431. Since we don't intend to support IE at this point, I'm closing this issue as wontfix.
